BraveRewardsUI/Tipping/TippingViewController.swift

```diff
@@ -72,9 +148,31 @@ class TippingViewController: UIViewController, UIViewControllerTransitioningDele
   }
   
   @objc private func tappedSendTip() {
-    tippingView.setTippingConfirmationVisible(true, animated: true)
-    DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) { [weak self] in
-      self?.dismiss(animated: true)
+    if let selectedIndex = self.tippingView.optionSelectionView.selectedOptionIndex {
+      let amount = self.tippingView.optionSelectionView.options[selectedIndex].value.doubleValue
+      
+      state.ledger.listRecurringTips { [weak self] info in
+        guard let self = self else { return }
+        
+        // If the user has recurring tips, remove the monthly tip
+        if info.firstIndex(where: { $0.id == self.publisherInfo.id }) != nil {
```

Comment:
Is this logic taken from brave-core?

Reply:
Uh I wrote it. I didn't take anything from Core. I took the information from the ticket and made the calls. To figure out which publishers have recurring tips, I made the only call that I saw had "recurringTips" and compared with our publisher to see if our publisher had tips. I couldn't find any fields for that.

Reply:
Im just not sure why you remove recurring tips here only just to re-add it?